.travis.yml

```diff
@@ -1,96 +1,60 @@
 language: cpp
+
+dist: trusty
```

Comment:
I assume it won't take years... but ok. I'm worried that we are now using the new distro, but because we made it explicit, over time we'll actually use the old distro.

But, I guess it's ok, as long as we keep updating this file.